dummy.env

```diff
@@ -2,3 +2,6 @@
 NODE_ENV=LOCAL
 # Disable by default in our open source environment
 AUTH0_ENABLED=false
+# Example of custom route adding
+CONTROLLER_NAMES=example1,example2
+CONTROLLER_URLS=http://example1:8080,http:example2:8080
```

Comment:
would it be better to make the env point to a json file (which is read in) that contains this data so that we do not have to write custom parsing and we do not have deal with improperly formatted data?

Reply:
hmmm, I see what you're saying, the tricky thing is we want this to be easily custom settable, we don't have a good way to custom load a json file into the gateway docker container - we could have the env var itself be json, but that also gets a bit tricky. This isn't the ideal solution but at least allows us to easy add some routes without changing the gateway docker container.